app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    print(parts)
+    source_path = parts[1]
```

Comment:
Make sure that the first part of the command is `cp` in the body of our function.